functests/utils/execute/discovery.go

```diff
@@ -0,0 +1,12 @@
+package execute
+
+import (
+	"os"
+	"strconv"
+)
+
+// DiscoveryModeEnabled indicates whether test discovery mode is enabled.
+func DiscoveryModeEnabled() bool {
```

Comment:
I am fine with having this as part of the `execute` package, but can we have it consistent with https://github.com/openshift/ptp-operator/pull/57 (changing the other is fine as well).
I hope one day, when everything is stable, we can collect all this util functions under a separate package